fvm/evm/handler/addressAllocator.go

```diff
@@ -73,13 +51,20 @@ func (aa *AddressAllocator) AllocatePrecompileAddress(index uint64) types.Addres
 }
 
 func MakePrecompileAddress(index uint64) types.Address {
-	return makePrefixedAddress(index, FlowEVMPrecompileAddressPrefix)
+	return makePrefixedAddress(index, types.FlowEVMExtendedPrecompileAddressPrefix)
 }
 
-func makePrefixedAddress(index uint64, prefix [addressPrefixLen]byte) types.Address {
+func makePrefixedAddress(
```

Comment:
I simplified the code here: https://github.com/onflow/flow-go/pull/5344. 
Please make sure the logic didn't change.